trin-types/src/consensus/header_proof.rs

```diff
@@ -29,12 +32,74 @@ pub struct BeaconChainBlockProof {
     pub slot: u64,
 }
 
-#[derive(Debug, PartialEq, Clone, Default)]
-pub struct HistoricalRoots(VariableList<tree_hash::Hash256, typenum::U16777216>);
+type HistoricalSummary = tree_hash::Hash256;
```

Comment:
Where does the name change `Root` -> `Summary` come from?

Reply:
`HistoricalSummary` is introduced in Cappella with the `historical_summaries` as `BeaconState` field which replaces the pre-Capella `hisotrical_roots` field.